packages/components/pagination/src/pagination.ts

```diff
@@ -429,7 +450,7 @@ export default defineComponent({
             ns.b(),
             ns.is('background', props.background),
             {
-              [ns.m('small')]: props.small,
+              [ns.m(String(_size!.value))]: true,
             },
```

Comment:
ns.m(_size.value),

Reply:
It seems that this hasn't changed. @webvs2